src/FileLoggerConfigurationLoader.java

```diff
@@ -0,0 +1,57 @@
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class FileLoggerConfigurationLoader {
+
+    public static FileLoggerConfiguration load(String filePath) {
+        BufferedReader reader = null;
+        String pathToFile = null;
+        LoggingLevel loggingLevel = null;
+        long maxSizeOfFile = 0;
+        String format = null;
+        try {
+            reader = new BufferedReader(new FileReader(filePath));
+            String line = reader.readLine();
+            while (line != null) {
+                String[] configurationParts = line.split(": ");
+                String value = configurationParts[1];
+                switch (configurationParts[0]) {
```

Comment:
Extract `configurationParts[0]` to a variable `String key = configurationParts[0]` and then use `key` in switch